packages/styles/src/mixins/_icons.scss

```diff
@@ -7,6 +7,17 @@
 @use './../functions/icons' as icon-fn;
 @use './../mixins/utilities';
 
+@mixin icon($name) {
+  background-color: currentColor;
```

Comment:
The idea behind using `currentColor` was to make the mask behave as if we were using a 'post-icon' component.
Opting for a fixed color will force us to manually declare a background color for each pseudo-state and transition on icons that need them instead of just inheriting them from the parent component. Don't you think ?